internal/packages/packageloader/transformfiles_funcs.go

```diff
@@ -0,0 +1,258 @@
+package packageloader
+
+import (
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
+)
+
+// allow all sprig functions except os, network and filepath.
+var allowedFuncNames = map[string]struct{}{
+	"hello": {},
+
+	// Dates
+	"ago":              {},
+	"date":             {},
+	"date_in_zone":     {},
+	"date_modify":      {},
+	"dateInZone":       {},
+	"dateModify":       {},
+	"duration":         {},
+	"durationRound":    {},
+	"htmlDate":         {},
+	"htmlDateInZone":   {},
+	"must_date_modify": {},
+	"mustDateModify":   {},
+	"mustToDate":       {},
+	"now":              {},
+	"toDate":           {},
+	"unixEpoch":        {},
+
+	// Strings
+	"abbrev":       {},
+	"abbrevboth":   {},
+	"trunc":        {},
+	"trim":         {},
+	"upper":        {},
+	"lower":        {},
+	"title":        {},
+	"untitle":      {},
+	"substr":       {},
+	"repeat":       {},
+	"trimAll":      {},
+	"trimSuffix":   {},
+	"trimPrefix":   {},
+	"nospace":      {},
+	"initials":     {},
+	"randAlphaNum": {},
+	"randAlpha":    {},
+	"randAscii":    {},
+	"randNumeric":  {},
+	"swapcase":     {},
+	"shuffle":      {},
```

Comment:
also random

Reply:
yes please kill all of them. Not totally related but https://reproducible-builds.org/ is a nice source (in my opinion) why that matters in general, just FYI :)

Reply:
see [here](https://github.com/package-operator/package-operator/pull/172#issuecomment-1431244223)